lib/data/user.dart

```diff
@@ -27,23 +27,23 @@ class Metadata {
     this.valid,
   });
 
-  Metadata.fromJson(Map<String, dynamic> json) {
-    pubkey = json['pub_key'];
-    name = json['name'];
-    displayName = json['display_name'];
-    picture = json['picture'];
-    banner = json['banner'];
-    website = json['website'];
-    about = json['about'];
-    if (json['nip05'] != null && json['nip05'] is String) {
-      nip05 = json['nip05'];
-    }
-    lud16 = json['lud16'];
-    lud06 = json['lud06'];
-    if (json['updated_at'] != null && json['updated_at'] is int) {
-      updated_at = json['updated_at'];
-    }
-    valid = json['valid'];
+  factory User.fromJson(Map<String, dynamic> json) {
+    final nip05 = json['nip05'];
+    final updated_at = json['updated_at'];
+    return User(
+      pubkey: json['pub_key'],
+      name: json['name'],
+      displayName: json['display_name'],
+      picture: json['picture'],
+      banner: json['banner'],
+      website: json['website'],
+      about: json['about'],
+      nip05: nip05 != null && nip05 is String ? nip05 : null,
+      lud16: json['lud16'],
+      lud06: json['lud06'],
+      updated_at: updated_at != null && updated_at is int ? updated_at : null,
```

Comment:
Weird, `dart analyze` wasn't offering to auto fix this variable name before, but now it is. Can you go ahead and make it camel case?